stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/client_endpoint.bal

```diff
@@ -72,6 +91,11 @@ public function <ClientEndpoint ep> init(ClientEndpointConfiguration config) {
     if (config.circuitBreaker != null) {
         ep.config = config;
         ep.httpClient = createCircuitBreakerClient(uri, config);
+    } else if (config.algorithm != null && lengthof config.targets > 1) {
+        ep.httpClient = createLoadBalancerClient(config);
+    } else if (config.failoverConfig != null) {
+        ep.config = config;
```

Comment:
this is unnecessary if you are setting the config inside `createFailOverClient()` right?

Reply:
it assigns config to the endpoint